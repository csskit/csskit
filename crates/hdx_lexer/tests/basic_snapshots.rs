mod helpers;

#[test]
fn basic_media() {
	assert_snap_tokens!("../../tasks/coverage/basic/media.css");
}

#[test]
fn basic_nth() {
	assert_snap_tokens!("../../tasks/coverage/basic/nth.css");
}

#[test]
fn basic_rule() {
	assert_snap_tokens!("../../tasks/coverage/basic/rule.css");
}

#[test]
fn basic_vars() {
	assert_snap_tokens!("../../tasks/coverage/basic/vars.css");
}
