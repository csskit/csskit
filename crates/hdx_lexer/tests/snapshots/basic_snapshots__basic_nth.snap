---
source: crates/hdx_lexer/tests/basic_snapshots.rs
expression: "get_tokens(\"../../tasks/coverage/basic/nth.css\".into())"
---
[
  {
    "kind": "Colon"
  },
  {
    "kind": "Function",
    "value": "nth-child"
  },
  {
    "kind": "Ident",
    "value": "n"
  },
  {
    "kind": "Number",
    "value": [
      1.0,
      {
        "bits": 16
      }
    ]
  },
  {
    "kind": "RightParen"
  },
  {
    "kind": "Whitespace"
  },
  {
    "kind": "LeftCurly"
  },
  {
    "kind": "RightCurly"
  },
  {
    "kind": "Whitespace"
  }
]
